src/Assingment1API.java

```diff
@@ -0,0 +1,107 @@
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Scanner;
+
+public class Assingment1API {
+
+    public static void main(String[] args) {
```

Comment:
can we separate the main class in another file and  findEmailAddressDetails, calculateAndPrintCarbonEmission in separate one?